draft-ietf-core-problem-details.md

```diff
@@ -111,64 +113,121 @@ non-empty<M> = (M) .and ({ + any => any })
+Consumers MUST use the type (tag number) as primary identifiers for
+the problem type; the "title" string is advisory and included only for
+consumers who are not aware of the semantics of the CBOR tag number
+used to indicate the specific problem type.
 
 The "detail" member, if present, ought to focus on helping the client correct the problem, rather than giving debugging information.  Consumers SHOULD NOT parse the "detail" member for information; extensions (see {{sec-new-attributes}}) are more suitable and less error-prone ways to obtain such information.
 
-Note that the "instance" URI reference may be relative; this means that it must be resolved relative to the document's base URI, as per {{!RFC3986}}.
+Note that the "instance" URI reference may be relative; this means
+that it must be resolved relative to the document's base URI, as per
+{{-uri}}.
+
+Note that the response code information that may be available together
+with a problem report is *not* replicated into a problem detail entry;
+compare this with "status" in {{-http-problem}}.
+
+{:aside}
+> (**Issue**: Do we still want to define a SPDK for status, so
+implementations can easily stash away the response code available from
+context into the problem details?)
 
 # Additional Problem Details
 {: #sec-new-attributes}
 
-To establish a custom problem type, A CBOR Tag number needs to be
+This specification defines a single problem type, the Generic Problem
+Details problem type (represented by CBOR tag TBD400, {{iana-tag}}).
+
+## Additional Problem Types
+
+To establish a new problem type, different from the Generic Problem
+Details problem type, a CBOR Tag number needs to be
 registered in the {{cbor-tags (CBOR Tags)<IANA.cbor-tags}} of {{!IANA.cbor-tags}}.
 Note that this registry allows the registration of new tags under the
 First Come First Served policy {{?RFC8126}}, making new registrations
 available in a simple interaction (e.g., via web or email) with IANA,
-having filled in the small template provided in {{Section 9.2 of
+after having filled in the small template provided in {{Section 9.2 of
 STD94}}.
+Such a registration SHOULD provide a documentation reference and also
+SHOULD reference the present specification.
+
+## Custom Problem Detail Entries
 
 Problem type definitions MAY extend the Problem Details document with
-additional entries to convey additional, problem-specific information.
-These receive a custom problem detail entry map key (unsigned integer
-or text)
-and SHOULD be described in a specification that goes along with such a
-registration.  Such a specification SHOULD also reference the present
-specification.
+additional entries to convey additional, problem-type-specific information,
+*custom problem details*.
+These receive a map key (custom problem detail entry map key, unsigned
```

Comment:
> These receive a map key

I suppose you don't mean exactly one, right? It should be fine that a custom type introduces multiple keys for its own namespace. Also the text two paragraphs below says

> In summary, the keys for Custom Problem Detail entries

Reply:
Thanks, clarified this a bit now.